recipes/recipes_emscripten/python/build.sh

```diff
@@ -2,7 +2,7 @@
 
 cp $RECIPE_DIR/patches/configure .
 
-PYTHON=${BUILD_PREFIX}/bin/python3.10
```

Comment:
I would revert the hard-coded changes and only change it in the python recipe but not **all** the other recipes